public/js/sidebar.js

```diff
@@ -16,10 +16,14 @@ export default class extends Component {
         this.downloadFastaOfSelected = this.downloadFastaOfSelected.bind(this);
         this.downloadAlignmentOfAll = this.downloadAlignmentOfAll.bind(this);
         this.downloadAlignmentOfSelected = this.downloadAlignmentOfSelected.bind(this);
+        this.componentDidMount = this.componentDidMount.bind(this);
```

Comment:
componentDidMount is an inbuilt react method that doesn't need to be bound (especially since you aren't particularly calling it yourself)

Reply:
Removed it from the constructor